### ISSUE TITLE
GPU is not work in mask RCNN

### ISSUE THREAD

**Issue by IdentifyResearchAreasandGoals** [matterport/Mask_RCNN#2865]
My enviroment is windows NVIDIA GeForce RTX3050LaptopGPU
•	Step 1: I  create Mask_RCNN  environment  python version is 3.6
	conda create -n Mask_RCNN python=3.6
•	 Step 2:Clone the repository. First, we will clone the mask rcnn repository which has the architecture for Mask R-CNN
Download mrcnn folder from this repository: https://github.com/matterport/Mask_RCNN
•	Step 3: Install the dependencies
	pip install tensorflow-gpu==1.15.0
	pip install keras==2.2.5
	pip install numpy
	pip install scipy
	pip install Pillow
	pip install cython
	pip install matplotlib
	pip install scikit-image
	pip install opencv-python
	pip install h5py==2.10.0
	pip install  IPython[all]
	python setup.py install
•	Step 4: Download the pre-trained weights (trained on MS COCO) ...
	Download coco weights : [https://github.com/matterport/Mask_RC.](https://www.youtube.com/redirect?
•	Step 5: Create folder : Dataset
	3- Create dataset folder. Under dataset folder -> Create 2 folders "train" and "val".
	4- Put training images in train folder.
	5- Put validation images in val folder
	6- Use VGGAnnotator tool for Annotations.

**Comment by IdentifyResearchAreasandGoals** [comment]
How to solve

**Comment by player3up** [comment]
same problem do u solve？

**Comment by QingMing114** [comment]
I solved the problem through this blog:
https://blog.csdn.net/qq_45743084/article/details/136207700
